plugins/Name_UpperCase.py

```diff
@@ -41,11 +41,14 @@ def init(self, logger):
             detail = T_(
 '''This feature is tagged with a name which contains a fully uppercase word (or words).
  This is not expected for the majority of named features.'''),
+            fix = T_(
+'''acronyms must be set in `short_name` tag.'''),
```

Comment:
I think acronyms should start with a capital `A`?

Also an optional suggestion to make it a bit more specific:
"Acronyms should be set in the `short_name` tag, unless the full name is never used in reality except for historic purposes".
As an example, a well-known general purpose store in The Netherlands is HEMA. Officially this means Hollandsche Eenheidsprijzen Maatschappij Amsterdam, but no-one in The Netherlands knows this full name (I had to look it up on Wikipedia as well to write this example); the former acronym is now the real name.